### DISCUSSION TITLE
Welcome to Fleet Discussions!

### DISCUSSION THREAD

**Discussion by DiegoDev2** [DiegoDev2/armada#33] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DrxcoDev** [comment]
¿Como se podrá solicitar la agregacion de algún paquete?

**Reply by DiegoDev2** [reply]
Aun no eh pensado eso, solo eh agregado formulas de paquetes.

**Comment by DiegoDev2** [comment]
y si tengo que implementar ese sistema gracias por recordar

**Reply by DrxcoDev** [reply]
De nada, lo digo porque tengo un par de paquetes ya creados.

**Reply by DiegoDev2** [reply]
si si si, entiendo.
